bin/codemods/run.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+const glob = require( 'glob' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	return glob.sync( 'bin/codemods/src/*.js' )
+		.map( name => path.basename( name, '.js' ) ) // strip path and extension from filename
+		.filter( name => name !== 'config' ); // exclude utility files
```

Comment:
It appears that `glob` allows for a reasonably powerful matching syntax - https://github.com/isaacs/node-glob

```js
return glob( 'bin/codemods/!(config).js' ).map( name => path.basename( name, '.js' ) )
```